cli_reference/helm_cli/configuring-custom-helm-chart-repositories.adoc

```diff
@@ -14,3 +14,5 @@ As a cluster administrator, you can add multiple Helm chart repositories, apart
 include::modules/helm-adding-helm-chart-repositories.adoc[leveloffset=+1]
 
 include::modules/helm-creating-credentials-and-certificates-to-add-helm-repositories.adoc[leveloffset=+1]
+
+include::modules/helm-disabling-helm-chart-repository-in-the-cluster.adoc[leveloffset=+1]
```

Comment:
Assemblies are supposed to meet discrete use goals, and removing the repo is the opposite of what the user would come to this assembly to do. I strongly suggest making a separate assembly to contain the module instead of adding it to this assembly.

Reply:
@kalexand-rh thank you for your feedback! I've committed all your suggestions and added a new assembly with a small introduction that now contains the modules. Please take a look when you have time. Thank you.